indicators/models.py

```diff
@@ -4,9 +4,10 @@
 from django.core.validators import MaxValueValidator, MinValueValidator
 from django.core.validators import RegexValidator
 
-__all__ = ['CountryIndicator', 'IndicatorField']
+__all__ = ['CountryIndicator', 'IndicatorField', 'INDICATORS_ORDER']
```

Comment:
In my opinion you can remove ```INDICATORS_ORDER```  from `__all__`

Reply:
It is going be used in another app (in generation results view)